apps/i18n/turtle/en_us.json

```diff
@@ -90,6 +90,8 @@
   "sizeParameter": "size",
   "step": "step",
   "tooFewColours": "You need to use at least %1 different colors for this puzzle.  You used only %2.",
+  "turnLeft": "turn left by",
```

Comment:
nit: can these keys be `turnLeftBy` and `turnRightBy`? It would be consistent with the variables in `blocks.js`.

Reply:
Good observation.  For the interim, it is "turnLeft" & "turnRight" as these are the strings that have translations in Crowdin, prior to the changes I made.  The follow-up PR (i.e to update all the draw_turn blocks with interpolateMsg) will only reference the "left" and "right" strings.

Reply:
Ah gotcha. Thanks for the explanation!